horace_core/sqw/@sqw/coordinates_calc.m

```diff
@@ -0,0 +1,189 @@
+function w = coordinates_calc(w, name)
+% Replace the sqw's signal and variance data with coordinate values (see below)
+%
+%   >> wout=coordinates_calc(w, name)
+%
+% Input:
+% -----
+%   w       Input sqw object
+%   name    Name of the parameter to use as the intensity
+%          Valid parameter names are:
+%               'd1','d2',...       Display axes (for as many dimensions as
+%                                  the sqw object has)
+%               'h', 'k', 'l'       r.l.u.
+%               'E'                 energy transfer
+%               'Q'                 |Q|
+%
+% Output:
+% -------
+%   w    Output sqw object with the signal and variance of the pixels set to the
+%          value of the input parameter name (also updates DND)
```

Comment:
really unclear what (also updates DND) means. You could mean that for DNDs, the image intensity is altered in this way; you could mean that for SQWs the DND component has its intensity altered; or both. Worth being detailed and explicit.